src/LtiLibrary.AspNetCore/Outcomes/v1/OutcomesControllerBase.cs

```diff
@@ -261,6 +261,16 @@ private static Result GetResult(ResultRecordType resultRecord)
             var replaceResponse = new replaceResultResponse();
 
             var result = GetResult(replaceRequest.resultRecord);
+
+            //The LTI 1.1 specification states in 6.1.1:
+            //The TC must check the incoming grade for validity and must fail when a grade is outside the range 0.0-1.0 or if the grade is not a valid number.   
+            //The TC must respond to these replaceResult operations with a imsx_codeMajor of "failure".
```

Comment:
Would you mind referencing the URL https://www.imsglobal.org/specs/ltiv1p1/implementation-guide#toc-26